pytorch/imagerecognition/openmpi-cifar10-resnet20-all-reduce-scaling/main.py

```diff
@@ -202,6 +213,14 @@ def main(run_id, dataset_dir, ckpt_run_dir, output_dir, validation_only=False,
                         help='Train with GPU')
     parser.add_argument('--light', action='store_true', default=False,
                         help='Train to light target metric goal')
+    parser.add_argument('--rank', type=str, default='1',
+                        help='The rank of the process')
+    parser.add_argument('--world_size', type=str, default='2',
+                        help='The world size')
```

Comment:
Maybe these should be `type=int`, as per the documentation of `init_process_group`:

world_size (python:int, optional) – Number of processes participating in the job. Required if store is specified.
rank (python:int, optional) – Rank of the current process. Required if store is specified.

Reply:
I fixed this with the commit. Thanks!